src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -287,6 +287,20 @@ public SuiteResult getPreviousResult() {
         return null;
     }
 
+     /**
+     * Check if all the tests exists for the suiteResult
+     * @param suiteResult
+     * @return true if all the tests were found, false otherwise
+     */
+    public boolean includesTestsFrom(SuiteResult suiteResult) {
```

Comment:
Or just make the method package-private. I see no particular reason to expose this into the API.